### ISSUE TITLE
Can't create room door cause of drinking machine but the other way round it is possible

### ISSUE THREAD

**Issue by ghost** [CorsixTH/CorsixTH#799]
It is possible to place a drinking machine near a door like this:

![bildschirmfoto von 2015-07-12 21 58 51](https://cloud.githubusercontent.com/assets/4523264/8639748/1e533872-28e3-11e5-8977-0704a4b5add8.png)

But it is not possible to create a door at the same place when the drinking machine is like this:

![bildschirmfoto von 2015-07-12 22 01 02](https://cloud.githubusercontent.com/assets/4523264/8639756/3c0319e6-28e3-11e5-875b-142b87097279.png)

expected:
- both should be possible or
- both shouldn't be possible

**Comment by TheCycoONE** [comment]
I believe this is the same missing check that allows building doors that are blocked by radiators.  Both shouldn't be possible.

**Comment by bobcanthelpyou** [comment]
In this case it could by a missing check by placing the drinking machine as i can't build a door near the drinking machine front?

**Comment by TheCycoONE** [comment]
Agreed.

**Comment by TheCycoONE** [comment]
Duplicate of #221
